mondrian/src/main/java/mondrian/rolap/SqlConstraintUtils.java

```diff
@@ -166,67 +166,61 @@ public static void addContextConstraint(
             // choose from agg or regular star
             String expr = getColumnExpr(sqlQuery, aggStar, column);
 
-            if ((RolapUtil.mdxNullLiteral().equalsIgnoreCase(value))
-                || (value.equalsIgnoreCase(RolapUtil.sqlNullValue.toString())))
-            {
-                sqlQuery.addWhere(expr, " is ", RolapUtil.sqlNullLiteral);
-            } else {
-                if (column.getDatatype().isNumeric()) {
-                    // make sure it can be parsed
-                    Double.valueOf(value);
-                }
+            if (column.getDatatype().isNumeric()) {
+                // make sure it can be parsed
+                Double.valueOf(value);
```

Comment:
If a numeric column contained a null value, this line of code would throw a number format exception.  Would this cause any issues?  For example, in steelwheels, you can update CUSTOMER_W_TER.CREDITLIMIT to contain a null value.

Reply:
You're right, my integration test validates that scenario.